sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/EcCryptographyClient.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    internal class EcCryptographyClient : ICryptographyProvider
```

Comment:
nit: this should probably be called EcCryptographyProvider not client, since it doesn't make any service calls.